be/src/exec/parquet_reader.cpp

```diff
@@ -152,6 +166,43 @@ Status ParquetReaderWrap::init_parquet_reader(const std::vector<SlotDescriptor*>
     }
 }
 
+Status ParquetReaderWrap::get_schema(std::vector<std::string>* col_names, std::vector<TypeDescriptor>* col_types) {
+    RETURN_IF_ERROR(_init_parquet_reader());
+
+    auto schema = _file_metadata->schema();
+    for (int i = 0; i < schema->num_columns(); ++i) {
+        auto column = schema->Column(i);
+        auto physical_type = column->physical_type();
```

Comment:
we need use both logical_type and physical_type, otherwise we can not get type like date、decimal

Reply:
Updated. The logical type is used to get timestamp type.

Reply:
I wonder if we can get logical_type from files written by old writers.

> ConvertedType is deprecated. However, to maintain compatibility with old writers, Parquet readers should be able to read and interpret ConvertedType annotations in case LogicalType annotations are not present.